src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -361,6 +361,28 @@ def load_arguments(self, _):
         c.argument('microsoft_account_client_secret', arg_group='Microsoft', help='AAD V2 Application client secret')
         c.argument('microsoft_account_oauth_scopes', nargs='+', help="One or more Microsoft authentification scopes (space-delimited).", arg_group='Microsoft')
 
+    with self.argument_context('webapp hybrid-connection') as c:
+        c.argument('name', arg_type=webapp_name_arg_type, id_part=None)
+        c.argument('slot', help="the name of the slot. Default to the productions slot if not specified")
+        c.argument('namespace', help="Hybrid connection namespace")
+        c.argument('hybrid_connection', help="Hybrid connection name")
+
+    with self.argument_context('functionapp hybrid-connection') as c:
+        c.argument('name', id_part=None)
```

Comment:
Why you want to turn off the `id` part. With it around you can get the `--ids` which you can supply a resource id of a webapp

Reply:
@singlessis you should make this change as a part of this PR